modules/core/import/modules/csv/tests/src/Kernel/AssetCsvImportTest.php

```diff
@@ -47,18 +49,27 @@ public function testAssetCsvImport() {
     $expected_values = [
       2 => [
         'name' => 'Old tractor',
+        'manufacturer' => '',
+        'model' => '',
+        'serial_number' => '',
```

Comment:
Should we provide a value for one of these bundle fields? Otherwise I don't think this test is demonstrating that bundle fields are correctly populated.

Reply:
Oops! Forgot to finish that. Good catch! I will push a fixup.